sender/kafka.go

```diff
@@ -0,0 +1,104 @@
+package sender
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/viper"
+	avro "github.com/topfreegames/avro/go/eventsgateway/generated"
+	"github.com/topfreegames/eventsgateway/forwarder"
+	pb "github.com/topfreegames/protos/eventsgateway/grpc/generated"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+type Kafka struct {
```

Comment:
Why public? NewKafka returns a Sender, so there doesn't seem to be a need to expose this.

Reply:
changed it to return *Kafka instead